.github/workflows/build-and-deploy.yml

```diff
@@ -98,7 +98,8 @@ jobs:
           echo "MINOR=$MINOR"
           echo "PATCH=$PATCH"
 
-          export DOCKER_BUILDKIT=1
```

Comment:
question: What's the reason for dropping BuildKit?

It was added to make the Dockerfile changes here work, I think. But it's the [more modern Docker way of building images](https://docs.docker.com/build/#overview), in any case. So maybe we still want to retain it?

We use it elsewhere; for example: https://github.com/opensafely-core/base-docker/blob/8d9de9f1c46816eca95b28f15e6aecc128f90b3e/Makefile#L4

Reply:
The reason is just as you think: we weren't using it before, and there isn't an immediate need to use it now, so we drop it. We may choose to use it when implementing #665, though.

Reply:
:+1: — I think it's fine to leave this PR as is, then semantically re-introduce BuildKit as its own improvement.